packages/runtime/runtime-utils/src/summaryUtils.ts

```diff
@@ -388,3 +389,78 @@ export class TelemetryContext implements ITelemetryContext {
 		return JSON.stringify(jsonObject);
 	}
 }
+
+/**
+ * Trims the leading slashes from the given string.
+ * @param str - A string that may contain leading slashes.
+ * @returns A new string without leading slashes.
+ */
+function trimLeadingSlashes(str: string) {
+	return str.replace(/^\/+/g, "");
+}
+
+/**
+ * Trims the trailing slashes from the given string.
+ * @param str - A string that may contain trailing slashes.
+ * @returns A new string without trailing slashes.
+ */
+function trimTrailingSlashes(str: string) {
+	return str.replace(/\/+$/g, "");
+}
+
+export class GCDataBuilder implements IGarbageCollectionData {
```

Comment:
Should this be `@internal` now that we moved it?

Reply:
The class. It does nothing functional, but at least it's an indication to potential consumers that this is not for them to use, so it gives us a bit more leeway if/when we need to change or remove it. Not everyone will notice, that's a risk, but at least we can say we're doing our best effort to communicate which things that are exported from a package are there just because **we** need to use them somewhere else, not because they're part of the "official" public API surface. And it's probably easier to justify a breaking change that affected a customer if it was on something that we indicated (not perfectly, but as best as we could) "you shouldn't use this".

Reply:
okay, I can do that